website/websitecore/templates/websitecore/case-study.html

```diff
@@ -0,0 +1,248 @@
+                    sodales
+                    aliquam. Mi hac
+                    sagittis orci id accumsan at. Vitae elementum viverra pellentesque risus erat tortor massa congue.
+                    Nunc
+                    scelerisque varius eu volutpat est praesent in. Augue eget in ante gravida nunc netus lorem
+                    suspendisse
+                    aliquam. Commodo eros, dui molestie at amet, eu fames vel arcu. Aliquam felis tellus, ultrices
+                    tortor
+                    mauris. Lorem ipsum dolor sit amet consectetur, adipisicing elit. Nam laboriosam excepturi
+                    aspernatur
+                    quaerat aliquid id animi. Lorem ipsum dolor sit amet consectetur
+                    adipisicing elit. Deleniti, quibusdam reprehenderit! Voluptatibus delectus sint inventore reiciendis
+                    a
+                    ipsum soluta distinctio at nobis. Cupiditate perferendis voluptates voluptatum accusamus aspernatur
+                    veritatis. In.
+                    <br>
+                    <br>
+                    <a class="btn-viewcase" href="">View Case Study</a>
+
+                </p>
+            </div>
+        </section>
+        <section class="bloc-contact-et-formulaire">
+            <div class="simplecontact">
+                <p class="firstp">Contact the team today for advice on your digital marketing campaign</p>
+                <p class="secondp">If you are looking to increase your traffic, sales & leads, let's have a chat!
+                </p>
+                <div class="call">
+                    <img class="mon-image" src="../../static/websitecore/img/case-study-images/grave.jpg"
+                        alt="mon image 30*30">
+                    <div>
+                        <p class="simplecontact-sms">Call or send a message <br><span class="tel">+237673488887</span>
+                        </p>
+
+                    </div>
+                </div>
+            </div>
+            <div class="formulaire">
+
+                <p class="titre-formulaire">Send us a message</p>
+
+                <p class="pform2">The team will usually respond within 1 hour.</p> <br>
+                <label for="name" aria-placeholder="yvan">Name</label>
+                <input class="name" type="text "> <br>
+                <br>
+                <label for="Email">Email Adress</label>
+                <input class="email" type="text"> <br>
+                <br>
+                <label for="phone">phone</label>
+                <input class="num" type="number" name="phone" id=""> <br>
+                <br>
+                <label for="Message">Message</label>
+                <input class="sms" type="Message" name="" id="aria-placeholder"> <br>
+                <br>
+                <input class="bouton" type="Submit" value="Submit">
+            </div>
+        </section>
+
+        <section class="navbarend">
```

Comment:
Tous cette partie doit etre dans le footer. 
The footer est sensé contenir le contenu qui cloture le document.